tests/python_package_test/test_dask.py

```diff
@@ -170,6 +222,32 @@ def test_regressor(output, client, listen_port):
     assert_eq(y, p2, rtol=1., atol=50.)
 
 
+@pytest.mark.parametrize('output', data_output)
+def test_regressor_pred_contrib(output, client, listen_port):
+    X, y, w, dX, dy, dw = _create_data('regression', output=output)
+
+    dask_regressor = dlgbm.DaskLGBMRegressor(
+        time_out=5,
+        local_listen_port=listen_port,
+        tree_learner='data'
+    )
```

Comment:
Add `n_estimators=10` and `num_leaves=10`?
#3786.

Reply:
added in https://github.com/microsoft/LightGBM/pull/3774/commits/64285895ec11c4a23af838c7cdfe62889a78b1a5

Reply:
added back again